infra/gcb/build_project.py

```diff
@@ -233,8 +233,13 @@ def get_build_steps(project_dir):
         if not workdir:
           workdir = '/src'
 
-        failure_msg = ('FAILED IN THE FOLLOWING CONFIG: --sanitizer {0} '
-                       '--engine {1}').format(sanitizer, fuzzing_engine)
+        failure_msg = ('*' * 80 + '\nFailed to build.\nTo reproduce, run:\n'
+                       'python infra/helper.py build_project {0}\n'
```

Comment:
nit:lets use named args, for anything > 1 argment

Reply:
Done!